.prettierrc.toml

```diff
@@ -6,9 +6,4 @@ trailingComma = 'all'
 [[overrides]]
   files = '*.js'
   [overrides.options]
-    trailingComma = 'es5'
-
-[[overrides]]
-  files = '*.{yml,yaml}'
-  [overrides.options]
-    tabWidth = 2
```

Comment:
This change has nothing to do with the task here and should have been done in a separate PR.